liquibase-core/src/main/java/liquibase/change/core/LoadDataChange.java

```diff
@@ -46,28 +46,7 @@
 import static liquibase.change.ChangeParameterMetaData.ALL;
 
 @DatabaseChange(name = "loadData",
-        description = "Loads data from a CSV file into an existing table. A value of NULL in a cell will be " +
```

Comment:
This information should be available somewhere. I agree it doesn't need to be in the JavaDocs but this is useful and valuable additional documentation for LoadDataChange.

Reply:
Thankfully the info is on the [docs page](https://docs.liquibase.com/change-types/load-data.html)!